js/index.js

```diff
@@ -1,3 +1,39 @@
 /* Logic for page animations and simple interactions should be placed here
    Any page or component specific logic should be placed in its own page/component js file.
-**/
\ No newline at end of file
+**/
+let image = [
+  "assets/img/IMG_8056.jpg",
+  "assets/img/IMG_8453.jpg",
+  "assets/img/IMG_7977.jpg",
+  "assets/img/IMG_7968.jpg"
+],
+currentPosition = 0;
+
+function carrousel(carrouselContainer) {
+carrouselContainer.addEventListener("click", e => {
+  let back = carrouselContainer.querySelector(".back"),
+    ahead = carrouselContainer.querySelector(".ahead"),
+    img = carrouselContainer.querySelector("img"),
```

Comment:
Are you sure that this is always going to get the same image element?

Reply:
I aint't understand the question

Reply:
If you use the tag `img` in the query selector, the program returns the first `img` that it finds; this could cause problems.
I recommend to use a class or an id with the querySelector.

Reply:
ahh ok, regarding your question, the image always changes and points to a higher or lower value as you can see in the conditional that is below the code
Even so, he told me that this is worth a talk explaining how it is done and how it works, so I can also clarify that in the talk.
Regarding your recommendation of a class or id, use both; the id for the js and the class for styles